app/views/pending_loans/show.html.erb

```diff
@@ -17,7 +17,7 @@
       <% @pending_loans.each do |project, loan_amount| %>
         <tr id=<%= "project_#{project.id}" %>>
           <td><%= project.tenant.organization %></td>
-          <td><%= link_to project.title, project_path(project) %></td>
+          <td><%= link_to project.title, tenant_project_path(slug: project.tenant.organization, id: project.id) %></td>
```

Comment:
The slug here should probably be "project.tenant.slug" (slug is almost the same as organization...it is the parameterized version, so there are cases such as org = "Save The Children" would have a slug "save-the-children".